scripts/rollup/forks.js

```diff
@@ -209,8 +209,6 @@ const forks = Object.freeze({
       entry === 'react-test-renderer'
     ) {
       return 'scheduler/src/forks/SchedulerHostConfig.mock';
-    } else if (entry === 'scheduler/unstable_post_task') {
-      return 'scheduler/src/forks/SchedulerHostConfig.post-task';
```

Comment:
Should we get rid of all the scheduler host configs eventually?

Reply:
Yeah it made more sense when the public API was larger and they all shared the same queue implementation. Going forward, looks more like the API will be small (just a few methods) and only some of them will use a userspace queue, as a fallback. So we can invert the relationship, get rid of the static injection.